cypress/integration/pages/storeSpec.ts

```diff
@@ -94,4 +91,21 @@ describe("Store", () => {
       });
     });
   });
+
+  describe("when awaiting labwares are in session storage", () => {
+    before(() => {
+      sessionStorage.setItem(
+        "awaitingLabwares",
+        "STAN-2111,tube,STAN-3111,Slide,STAN-4111,Slide,STAN-5111,Slide"
+      );
+      cy.visit("/store");
+    });
+
+    it("store all button should be enabled", () => {
+      cy.findByRole("button", { name: /Store All/i }).should("be.disabled");
```

Comment:
The test name is "should be enabled" but the test says should("be.disabled")

Reply:
You are right. The test name should be changed and I have updated it.